GitHubReactions.ps1

```diff
@@ -0,0 +1,363 @@
+    if ($PSBoundParameters.ContainsKey('PassThru')) {
+        return $result
+    }
+}
+
+function Remove-GitHubReaction
+{
+<#
+    .SYNOPSIS
+        Removes a reation on a given GitHub issue.
+
+    .DESCRIPTION
+        Removes a reation on a given GitHub issue.
+
+        The Git repo for this module can be found here: http://aka.ms/PowerShellForGitHub
+
+    .PARAMETER OwnerName
+        Owner of the repository.
+        If not supplied here, the DefaultOwnerName configuration property value will be used.
+
+    .PARAMETER RepositoryName
+        Name of the repository.
+        If not supplied here, the DefaultRepositoryName configuration property value will be used.
+
+    .PARAMETER Uri
+        Uri for the repository.
+        The OwnerName and RepositoryName will be extracted from here instead of needing to provide
+        them individually.
+
+    .PARAMETER Issue
+        The issue number.
+
+    .PARAMETER ReactionId
+        The Id of the reaction. You can get this from using Get-GitHubReaction.
+
+    .PARAMETER AccessToken
+        If provided, this will be used as the AccessToken for authentication with the
+        REST Api.  Otherwise, will attempt to use the configured value or will run unauthenticated.
+
+    .PARAMETER NoStatus
+        If this switch is specified, long-running commands will run on the main thread
+        with no commandline status update.  When not specified, those commands run in
+        the background, enabling the command prompt to provide status information.
+        If not supplied here, the DefaultNoStatus configuration property value will be used.
+
+    .EXAMPLE
+        Remove-GitHubReaction -OwnerName PowerShell -RepositoryName PowerShell -Issue 12626 -ReactionId 1234
+
+        Remove a reaction by Id on Issue 12626 from the PowerShell\PowerShell project.
+
+    .EXAMPLE
+        Get-GitHubReaction -OwnerName PowerShell -RepositoryName PowerShell -Issue 12626 -ReactionType rocket | Remove-GitHubReaction
+
+        Gets a reaction using Get-GitHubReaction and pipes it into Remove-GitHubReaction.
+
+    .NOTES
+        Currently, this only supports reacting to issues. Issue comments, commit comments and PR comments will come later.
+#>
+    [CmdletBinding(
+        SupportsShouldProcess,
+        DefaultParameterSetName='Elements')]
+    [Diagnostics.CodeAnalysis.SuppressMessageAttribute("PSShouldProcess", "", Justification="Methods called within here make use of PSShouldProcess, and the switch is passed on to them inherently.")]
+    param(
+        [Parameter(Mandatory, ParameterSetName='Elements', ValueFromPipelineByPropertyName)]
+        [string] $OwnerName,
+
+        [Parameter(Mandatory, ParameterSetName='Elements', ValueFromPipelineByPropertyName)]
+        [string] $RepositoryName,
+
+        [Parameter(Mandatory, ParameterSetName='Uri')]
+        [string] $Uri,
+
+        [Parameter(Mandatory, ParameterSetName='Elements', ValueFromPipelineByPropertyName)]
```

Comment:
There's no need to specify which parameter set this is mandatory in if it's mandatory in all of them.  Here and below.  Specifying a specific parameter set _will_ be necessary once you start adding support for additional reaction types.

```suggestion
        [Parameter(Mandatory, ValueFromPipelineByPropertyName)]
```

Reply:
Fixed.